apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -308,21 +308,59 @@ describe('FinishStudentAccount', () => {
     expect(screen.queryByText(locale.email_error_message())).toBe(null);
     expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
 
-    // Enter state
+    // Enter parent email
     fireEvent.change(parentEmailInput, {target: {value: 'parent@email.com'}});
 
     // Error does not show and button is enabled when email is entered
     expect(screen.queryByText(locale.email_error_message())).toBe(null);
     expect(finishSignUpButton.getAttribute('aria-disabled')).toBe(null);
 
-    // Clear state
+    // Clear parent email
     fireEvent.change(parentEmailInput, {target: {value: ''}});
 
     // Error shows and button is disabled with empty email
     screen.getByText(locale.email_error_message());
     expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
   });
 
+  it('parentEmail error shows if parent checkbox is selected and parentEmail is an invalid email', async () => {
+    renderDefault();
+    await waitFor(() => {
+      expect(fetchStub.calledOnce).toBe(true);
+    });
+    const finishSignUpButton = screen.getByRole('button', {
+      name: locale.go_to_my_account(),
+    });
+
+    // Set other required fields
+    const displayNameInput = screen.getAllByDisplayValue('')[1];
+    const stateInput = screen.getAllByRole('combobox')[1];
+    const ageInput = screen.getAllByRole('combobox')[0];
+    fireEvent.change(displayNameInput, {target: {value: 'FirstName'}});
+    fireEvent.change(stateInput, {target: {value: 'WA'}});
+    fireEvent.change(ageInput, {target: {value: '6'}});
+
+    // Button is enabled after required fields are filled before parent checkbox is checked
+    expect(finishSignUpButton.getAttribute('aria-disabled')).toBe(null);
```

Comment:
nit pick, but if you want to test both `aria-disabled` and the `disabled` attribute at the same time, you can do `expect(finishSignUpButton).not.toBeDisabled();`, which is a matcher built into react testing library

Reply:
Omg that's a game changer thank you! I'll update all of them with this!